wind_poses:
  traj1:
    gripper: 0.95
    poses:
      pose1:
        position: 
          x: -0.11096636168682528
          y: -0.04895707368242652
          z: -0.0007318817079067369
        orientation: 
          x: 0.7232567170479167
          y: 0.6905742382545376
          z: 0.0015137915337679873
          w: -0.002156650047559387

      pose2:
        position: 
          x: -0.10240
          y: 0.049946
          z: -0.00070
        orientation: 
          x: 0.7232903282738873
          y: 0.6905390796985692
          z: 0.0015044351487119188
          w: -0.002148746003894388

  traj2:
    gripper: 1.0
    poses:
      pose3:
        position: 
          x: -0.01849
          y: 0.08323
          z: 0.09836
        orientation: 
          x: 0.9020339021872683
          y: 0.43166181535425785
          z: 0.0011239624035841505
          w: -0.0012857598500842869

      pose4:
        position: 
          x: -0.0358036472457251
          y: -0.053651313593225225
          z: 0.09614058583974838
        orientation: 
          x: 0.9035982510680952
          y: 0.4283708320589584
          z: 0.00223048919382015
          w: -0.0019120213784429075


  traj3:
    gripper: 0.95
    poses:
      pose5:
        position: 
          x: -0.018302062172802345
          y: -0.06469275509300482
          z: 0.09219894930720328
        orientation: 
          x: 0.3234914251967029
          y: 0.9462013609290655
          z: 0.004910813461909913
          w: -0.005671535183073252

      pose6:
        position: 
          x: -0.0725950705517956
          y: -0.15299484609086045
          z: 0.09336031064391135
        orientation: 
          x: 0.2752823796258717
          y: 0.961292061222201
          z: 0.0065182090904140145
          w: -0.009731261426103189
     
      pose7:
        position: 
          x: -0.030233446742947062
          y: -0.24851968169729316
          z: 0.04110465198755264
        orientation: 
          x: 0.27525493440721627
          y: 0.9612993193067394
          z: 0.006525689237598234
          w: -0.0097854568034995

  traj4:
    gripper: 1.0
    poses:
      pose8:
        position: 
          x: -0.0816980179724342
          y: -0.23888139586181817
          z: -0.028249330818653107
        orientation: 
          x: 0.4738171387751731
          y: 0.8805773636248525
          z: 0.004130177733854237
          w: -0.00798544340029533

      pose9:
        position: 
          x: -0.12248845179517301
          y: -0.15108988837449028
          z: -0.024347424507141113
        orientation: 
          x: 0.7231141845098621
          y: 0.6906883439462825
          z: 0.001976103409194794
          w: -0.007182110601693866


  traj5:
    gripper: 0.95
    poses:
      pose10:
        position: 
          x: -0.11096636168682528
          y: -0.04895707368242652
          z: -0.0007318817079067369
        orientation: 
          x: 0.7232567170479167
          y: 0.6905742382545376
          z: 0.0015137915337679873
          w: -0.002156650047559387

      pose20:
        position: 
          x: -0.06484437710930316
          y: 0.06856509427905699
          z: -0.0006782263517379761
        orientation: 
          x: 0.7232903282738873
          y: 0.6905390796985692
          z: 0.0015044351487119188
          w: -0.002148746003894388

  traj6:
    gripper: 1.0
    poses:
      pose30:
        position: 
          x: -0.02199
          y: 0.08323
          z: 0.09836
        orientation: 
          x: 0.9020339021872683
          y: 0.43166181535425785
          z: 0.0011239624035841505
          w: -0.0012857598500842869

      pose40:
        position: 
          x: -0.0358036472457251
          y: -0.053651313593225225
          z: 0.09614058583974838
        orientation: 
          x: 0.9035982510680952
          y: 0.4283708320589584
          z: 0.00223048919382015
          w: -0.0019120213784429075


  traj7:
    gripper: 0.95
    poses:
      pose50:
        position: 
          x: -0.018302062172802345
          y: -0.06469275509300482
          z: 0.09219894930720328
        orientation: 
          x: 0.3234914251967029
          y: 0.9462013609290655
          z: 0.004910813461909913
          w: -0.005671535183073252

      pose60:
        position: 
          x: -0.0725950705517956
          y: -0.15299484609086045
          z: 0.09336031064391135
        orientation: 
          x: 0.2752823796258717
          y: 0.961292061222201
          z: 0.0065182090904140145
          w: -0.009731261426103189
     
      pose70:
        position: 
          x: -0.0404
          y: -0.2453
          z: 0.0411
        orientation: 
          x: 0.27525493440721627
          y: 0.9612993193067394
          z: 0.006525689237598234
          w: -0.0097854568034995

  traj8:
    gripper: 1.0
    poses:
      pose80:
        position: 
          x: -0.09345877588196094
          y: -0.24527690203937175
          z: 0.0016538165509700636
        orientation: 
          x: 0.4738171387751731
          y: 0.8805773636248525
          z: 0.004130177733854237
          w: -0.00798544340029533

      pose90:
        position: 
          x: -0.1697720625034373
          y: -0.1427179953502044
          z: -0.017504300922155394
        orientation: 
          x: 0.7231141845098621
          y: 0.6906883439462825
          z: 0.001976103409194794
          w: -0.007182110601693866



# pose3 >round 2:[-0.011055756739748901, 0.07915848698323102, 0.09835994988679886, 0.9020272058695236, 0.4316757143796672, 0.0011292765141215607, -0.001312334019583212]  
